document/CoreFormat.mdk

```diff
@@ -183,113 +179,22 @@ and contains the output of that run.
 ```
 ~
 
-### `toolInfo` property { #runLog-toolInfo }
```

Comment:
This information hasn't gone away. What's really happened is that the information from below that was previously under `run` (the former `runInfo`) has moved upwards in the doc to live directly under the new `run` (the former `runLog`). This will be easy for you to see in the HTML.